README.md

```diff
@@ -68,11 +69,28 @@ The root schemadocument is called *ebutt_live.xsd*.
 The Python library
 ==================
 
-The library uses XSD schemas from the xsd1.1 subdirectory. The bindings are auto-generated by PyXB.
-For binding generation one can use the following code:
-
-    pyxbgen -r --binding-root=./ebu_tt_live/bindings -m __init__ --schema-root=./ebu_tt_live/xsd1.1/ -u ebutt_live.xsd
```

Comment:
because the makefile has the line. Also couple lines above that the same line is present.